csruby_frontend_app/src/components/traders/TraderPreview.js

```diff
@@ -13,22 +17,23 @@ class TraderPreview extends Component {
       username: props.username,
       created_at: props.createdAt,
       action: props.action,
-      belongsToAuthedUser: false,
       response_description: '',
     }
+
+    this.closeButtons = document.getElementsByClassName('refresh-on-close');
+
   }
 
   componentDidMount() {
-    if(this.state.authed_user === this.state.email) {
-      this.setState({
-        belongsToAuthedUser: true
-      });
-    }
-
-    if(this.state.authed_user === null) {
-      this.setState({
-        belongsToAuthedUser: false
-      });
+    this.closeButtons = document.getElementsByClassName('refresh-on-close');
+    // TODO : refresh la fenêtre après la fermeture de la boîte modale
+    // console.log("HERE");
+    // console.log(this.closeButtons);
+    // console.log(this.closeButtons.length;
+    for (let button of this.closeButtons) {
+      // console.log(button);
+      // console.log("DANS BOUCLE");
+      button.addEventListener('click', event => {console.log("closing");window.location.reload();});
```

Comment:
![image](https://user-images.githubusercontent.com/55529471/77822677-35322a80-70f5-11ea-8558-56384238812b.png)

Remove the fookin comments